game/templates/game/board.html

```
@@ -537,40 +411,84 @@
-        
-<!-- Right Panel: Controls -->
-        <div class="panel">
-            <div class="card">
+
+        <!-- Consolidated Sidebar -->
+        <div class="sidebar-container" id="mobilePanel">
+            <button type="button" id="mobileControlsClose" aria-label="Close controls" class="mobile-close-btn">&times;</button>
+            <div class="card status-card">
+                <div class="status-bar" id="statusBar">
+                    <div id="replayControls" class="replay-controls hidden" aria-label="Game Replay Controls" role="group">
+                        <button id="firstReplayBtn" aria-label="Go to first move">⏮</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)